### ISSUE TITLE
Update operational cadence

### ISSUE THREAD

**Issue by jlusds** [CDCgov/prime-reportstream#1777]
Figure out new operational cadence - “for a given state that is onboarding, we have the ability to provide an update every 2 days”

**Comment by Adrian-Brewster** [comment]
https://cdc-my.sharepoint.com/:w:/g/personal/mpz9_cdc_gov/EWkOJRh2StxFqta7FMmHSqABo7NNPsR6Vozx_cfys406Sw?email=qpu0%40cdc.gov&e=n3eHMV

State Operations – Status Update Cadence 
 
Current Workflow:  
 - James updates tracker (in teams) 2-3x per week from email or other updates received via slack 
 - James tags people in Slack when a specific action item is assigned to them  
 - Updates that need to be provided to states are divvied up between James and Mo  
           - This is communicated via slack 
 
Issues: 
 - Process is informal 
 - States are only receiving updates once a week currently 
 
 
New Proposed Workflow: 
 - All team members are responsible for updating the tracker as updates come in.  If you respond to an email, you are responsible for      capturing what was discussed in that email in the tracker. This increases visibility for all team members and keeps the updates in real time. 
 - There is a single key POC for each state.  That person is responsible for providing all updates to their assigned states.  This does not change week to week unless someone is out of office.  
 - Action items needs to be tracked in GitHub (or another issue tracker?) and have a single assigned person.  
 - States will receive updates 3 times a week (M/W/F) instead of once a week. 
                - We may be able to automate this process a bit using SendGrid.  Will need to meet with Drew and Matt to discuss.

**Comment by Adrian-Brewster** [comment]
State Operations – Status Update Cadence 
 
Context: 
ReportStream is growing, rapidly. As we onboard additional senders and our overall message volume increases, we need to consider how we can operationalize ongoing support and communications with our receiving state jurisdictions.  In order to stay current with ticket requests, onboarding activities, and various email conversations (while maintaining visibility for critical stakeholders) we need to reconsider how we organize responsibilities and data flow on this project.  
 
Key Considerations/Dependencies: 
Sparkle and Ming need visibility across both ReportStream and SimpleReport. Because of the work they are doing on SimpleReport, they are often in communication with these states and may have some valuable insight/context to share prior to the ReportStream team initiating communications with the state.  
Solution #1: Further separate ReportStream and SimpleReport to function siloed and autonomously from one another. Communications from each team are handled separately. We need to make sure we are not duplicating effort so division of responsibilities will need to be clear but we can not have one team roadblocked or waiting on the other. We will continue having a weekly standup to align goals for the sprint and make sure each team is clear on what they are responsible for.  
Solution #2: We increase visibility across the program by better utilizing our Jurisdictional Tracker with updates (and eventually the new CRM). Anytime there is a pertinent email conversation that can add context to a specific state, a note needs to be added to the tracker.  This way when a resource is ready to engage a state, they simply review the notes from both the RS and SR teams. This eliminates the need to run communications past multiple teams. The tracker will be considered the single source of truth.  
Resource limitations require us to “do less” in an attempt to keep our level of customer service up.  States are used to a lot of hand holding.  Because we ultimately need to advocate for the states and make this connection as easy and low maintenance for them as possible, we need to prioritize individual attention over expediency.  This may mean engaging fewer states at a time or reprioritizing some enhancements each sprint. When considering the typical project management prioritization triangle if time, money, and quality, we need to prioritize quality.   
 
 We need to create a workflow that is not dependent on any single person. If someone is travelling or reallocated to another project quickly, we need to be able to accommodate without it being too disruptive.   
 
Current Workflow:  
James updates tracker (in teams) 2-3x per week from email or other updates received via slack 
James tags people in Slack when a specific action item is assigned to them  
Updates that need to be provided to states are divvied up between James and Mo  
This is communicated via slack 
Very informal/ad hoc 
Little visibility across the program.  
Process is dependent on individuals instead of teams.  
 
 
New Proposed Workflow: 
 
All team members are responsible for updating the tracker as updates come in.  If you respond to an email, you are responsible for capturing what was discussed in that email in the tracker. This increases visibility for all team members and keeps the updates in real time. 
There is a single key POC for each state.  This person is identified at the sprint kick off and will be responsible for tracking progress from planning to production.  This person will likely need to be technical, so I suggest Brick, Mo, and Jorge each get assigned 1-2 states per sprint.  Adrian and James can help facilitate conversations as needed.  
Action items need to be tracked in GitHub (or another issue tracker?) and have a single assigned person.  
States will receive updates 3 times a week (M/W/F) instead of once a week. 
We may be able to automate this process a bit using SendGrid.  Will need to meet with Drew and Matt to discuss.  
These updates will include the following sections: 
Work completed 
Work assigned (with individual assignments for every action item included) 
Any blockers 
Timeline expectations will be provided to the states prior to kick off.  
 
 
 
Ongoing State Operations: 
 Once a state achieves go live status, we need to determine the cadence for outreach.  States will need to be notified when a new sender has onboarded to ReportStream so that their messages can undergo proper validation.   
States will receive weekly email updates that include all of the SimpleReport sites 
A biweekly newsletter will be sent to all jurisdictions to provide highlights and updates from all things related to ReportStream.  
We will need to build out a ticketing system for states to submit issue tickets.  This can be as simple as creating a helpdesk email.
